isaric/parsers/isaric-core.toml

```diff
@@ -0,0 +1,2546 @@
+    ref = "Y/N/NK"
+
+[subject.has_malignant_neoplasm]
+  field = "malignantneo_mhyn"
+  description = "<acronym title=\"Current solid organ or haematological malignancy. Please do not include malignancy that have been declared 'cured' >=5 year ago with no evidence of ongoing disease. Do not include non-melanoma skin cancers. Do not include benign growth or dysplasia.\">Malignant neoplasm</acronym>"
+  ref = "Y/N/NK"
+
+[subject.has_malnutrition]
+  field = "malnutrition_mhyn"
+  description = "<acronym title=\"Any clinically identified deficiency in intake, either of total energy or of specific nutrient that led to a dietetic intervention or referral prior to the onset of COVID-19 symptoms. Do not include people who needed supplementary nutrition solely due to reduced intake during their current illness episode.\">Malnutrition</acronym>"
+  ref = "Y/N/NK"
+
+[subject.has_obesity]
+  field = "obesity_mhyn"
+  description = "<acronym title=\"This refers to patient for whom an attending clinician ha assessed them to be obese - ideally but not necessarily with an objective measurement of obesity, such a calculation of the body mass index (BMI of 30 or more) or measurement of abdominal girth.\">Obesity <i>(as defined by clinical staff)</i></acronym>"
+  ref = "Y/N/NK"
+
+[subject.has_rheumatologic_disorder]
+  field = "rheumatology_mhyr"
+  description = "<acronym title=\"This is defined a an inflammatory and degenerative disease of connective tissue structures. It includes chronic arthropathy and arthritis, connective tissue disorder and vasculitides.\">Rheumatologic disorder</acronym>"
+  ref = "Y/N/NK"
+
+[subject.has_smoking]
+  field = "smoking_mhyn"
+  description = "<acronym title=\"Smoking at least one cigarette, cigar, pipe or equivalent per day before the onset of the current illness. Do not include smoke-free tobacco product such a chewed tobacco or electronic nicotine delivery devices.\">Smoking</acronym>"
+
+  [subject.has_smoking.values]
+    2 = "never"
+    3 = "former"
+
+[subject.has_tuberculosis]
+  field = "tb_mhyn"
+  description = "<acronym title=\"Patients currently receiving treatment for tuberculosis. Do not include latent tuberculosis.\">Tuberculosis</acronym>"
+  ref = "Y/N/NK"
+
+[subject.icu_admitted]
+  combinedType = "any"
+
+  [[subject.icu_admitted.fields]]
+    field = "flw2_icu"
+    description = "If ever admitted to hospital, were you admitted to intensive care (ICU/ITU)?"
+
+    [subject.icu_admitted.fields.values]
+      1 = true
+      0 = false
+
+  [[subject.icu_admitted.fields]]
+    field = "flw_icu"
+    description = "If ever admitted to hospital, were you admitted to intensive care (ICU/ITU)?"
+
+    [subject.icu_admitted.fields.values]
+      1 = true
+      0 = false
+      2 = "not sure"
+
+  [[subject.icu_admitted.fields]]
+    field = "ccm_b_icu_hodat"
+    description = "<acronym title=\"Date admitted to critical care ward\">ICU ADMISSION DATE</acronym>"
+
```

Comment:
These two are date fields ... are they suitable to check if someone was admitted to icu? .. or should we assume if the field empty, it indirectly says the person was not admitted to icu?

Reply:
Thanks - forgot to add an apply function to check if data has been entered (and therefore ICU admission = True)